db/psqlPool.js

```diff
@@ -0,0 +1,29 @@
+const { Pool } = require('pg');
+require('dotenv').config();
+
+let pool; 
+if (process.env.NODE_ENV === 'production') {
+  pool = new Pool({
+    user: process.env.PROD_DB_USR,
+    host: process.env.PROD_HOST,
+    database: process.env.PROD_DB,
+    password: process.env.PROD_DB_PW,
+    ssl: true,
+    port: 5432,
+  });
+} else {
+  console.log(process.env.DEV_DB_PW);
```

Comment:
Lets remove the console logs that exposes the environment. Not great practice.

Reply:
Resolved: Removed console logs that exposed environment